src/main/java/org/randywebb/wlts/ldstools/rest/LdsToolsClient.java

```diff
@@ -56,20 +56,20 @@ public LdsToolsClient(String user, String password) throws AuthenticationExcepti
     }
 
     /**
-     * Deliberately package scoped to enable other ldstools api components
+     * Deliberately package scoped to enable other ldstools api components.
```

Comment:
Comment says it is deliberately package scoped, but line 64 shows that we changed it to public at some point. Update (remove) the comment accordingly?